frontend/src/graphql-components/apolloClientSetup/ApolloClientSetup.tsx

```diff
@@ -65,7 +68,7 @@ const ApolloClientSetup: FC<PropsWithChildren<{}>> = props => {
   }, [token]);
   return (
     <>
-      {isLoggedIn && apolloClient && (
+      {(isLoggedIn || hasRenderingError(errorsQueue)) && apolloClient && (
```

Comment:
Please use parentheses to wrap conditions here: mixing && and || together can be misleading (and even not working as expected).

Reply:
I've just noticed that `hasRenderingError()` is not necessary there.